tests/testthat/cubes/cat-x-cat.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
These look like the same cube fixtures that are in `crunch/tests/testthat/cubes/`, so maybe those should be moved to `crunch/inst/` or something so they can be shared and not duplicated. The advantage of that would be that we wouldn't forget to update this second copy of the fixtures if/when the cube response API evolves.

Reply:
We decided to put this off, I raised an issue about it in rcrunch.